java/ql/src/Security/CWE/CWE-522/InsecureLdapAuth.ql

```diff
@@ -0,0 +1,25 @@
+/**
+ * @name Insecure LDAP authentication
+ * @description LDAP authentication with credentials sent in cleartext.
```

Comment:
```suggestion
 * @description LDAP authentication with credentials sent in cleartext makes sensitive information vulnerable to remote attackers
```
Just suggest updating this description to follow the "Syntax X causes behavior Y" pattern, as mentioned in the [style guide](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description). Feel free to change my suggestion if it is not accurate!